lib/index.js

```diff
@@ -55,29 +53,10 @@ Chameleon.prototype.loaded = function() {
 Chameleon.prototype.identify = function(identify) {
   var options = identify.traits();
 
-  options.uid = options.id || identify.userId() || identify.anonymousId();
+  var uid = options.id || identify.userId() || null;
   delete options.id;
 
-  window.chmln.identify(options);
-};
-
-/**
- * Associate the current user with a group of users.
- *
- * @api public
- * @param {Facade} group
- */
-
-Chameleon.prototype.group = function(group) {
```

Comment:
@hankim813 Those functions were no-ops on the backend for us -- I'll add them back with similar implementation on the frontend.